akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -877,39 +637,39 @@ final class GraphInterpreter(
    */
   def dumpWaits(): Unit = println(toString)
 
-  override def toString: String = {
```

Comment:
this is useful debug info, but the assembly is no longer there so it needs to be fixed. I think this also relates to https://github.com/akka/akka/issues/22433